build/tutorials/tutorials-contest/red-dev-avalanche-chainlink-tutorial/ansible-chainlink-avalanche-setup/README.md

```diff
@@ -0,0 +1,19 @@
+# reddev infrastructure automation repository
```

Comment:
[Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022)

Reply:
This is done. @cgcardona